src/ast/ddl.rs

```diff
@@ -1120,6 +1120,13 @@ pub enum ColumnOption {
     /// [1]: https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#view_column_option_list
     /// [2]: https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#column_option_list
     Options(Vec<SqlOption>),
+    /// MS SQL Server specific: Creates an identity column in a table.
+    /// Syntax
+    /// ```sql
+    /// IDENTITY [ (seed , increment) ]
+    /// ```
+    /// [MS SQL Server]: https://learn.microsoft.com/en-us/sql/t-sql/statements/create-table-transact-sql-identity-property
+    Identity(Option<SqlOption>),
```

Comment:
Is this correct to have a `SqlOption` in the identity? Thinking this would allow a lot of invalid identity values where the only possible value from the docs seems to be a tuple so `Identity(Option<(Value, Value)>)`? That way we don't have to add `Identity` both here as a `ColumnOption` and as a new `SqlOption`

Reply:
Thanks for the suggestion!

You’re correct that `Identity(Option<(Value, Value)>)` would potentially prevent invalid identity values. However, implementing `Identity(Option<(Value, Value)>)` as proposed presents some challenges. Specifically, the traits `ast::visitor::VisitMut` and `ast::visitor::Visit` are not implemented for `(ast::value::Value, ast::value::Value)`, which makes it difficult to use in this form.

An alternative approach could be to introduce a dedicated struct like:

```rust
pub struct IdentityColumn {
    pub seed: Value,
    pub increment: Value,
}
```
And then use it as `Identity(Option< IdentityColumn>)`.

Let me know what you think!

Reply:
I think that sounds like a better idea compared to adding the variant to `SqlOption` but maybe @iffyio have some input before you go that route?

Reply:
Yeah having it as a dedicated type sounds reasonable (maybe call it `IdentityProperty` instead of column since the former is mentioned in the docs - I'm guessing the column is more of the effect)?

Reply:
Thank you for suggestions! I updated it with a dedicated type `IdentityProperty`